t3f/ops.py

```diff
@@ -802,20 +802,26 @@ def frobenius_norm_squared(tt, differentiable=False):
     sum of squares of all elements in `tt`
   """
   if differentiable:
+    if hasattr(tt, 'batch_size'):
+        bs_str = "n"
```

Comment:
Please, use single quotes as in the rest of the files (I don't care, but consistency is good)